src/gen.sh

```diff
@@ -2,5 +2,9 @@
 #Concatenate files into one services file
 SRCDIR=$(dirname $0)
 cd "$SRCDIR"
+
+# Delete old build files
+rm -r ../build/
```

Comment:
does 'make clean' not do this? or you want to run it each time?

Reply:
No. 
make clean do:
rm -f build/foam2.jar
rm -f build/foam2.classpath.txt
rm -rf .foam2-gensrcs

Reply:
so why not add this to make clean?

Reply:
with the clean change above, this is no longer needed correct?